packages/account/src/Sections/Verification/ProofOfIncome/__tests__/proof-of-income.spec.tsx

```diff
@@ -0,0 +1,174 @@
+import React from 'react';
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { WS } from '@deriv/shared';
+import { StoreProvider, mockStore } from '@deriv/stores';
+import ProofOfIncome from 'Sections/Verification/ProofOfIncome';
+
+const mocked_get_account_status = {
+    authentication: {
+        attempts: { latest: { service: '' }, history: [] },
+        document: {
+            status: 'none',
+        },
+        identity: {
+            status: 'none',
+            services: { idv: {}, onfido: {}, manual: {} },
+        },
+        income: {
+            status: 'none',
+        },
+        needs_verification: [],
+    },
+    status: [],
+};
+
+jest.mock('@binary-com/binary-document-uploader');
+jest.mock('Components/demo-message', () => jest.fn(() => 'MockedDemoMessage'));
+jest.mock('@deriv/shared', () => ({
+    ...jest.requireActual('@deriv/shared'),
+    WS: {
+        authorized: {
+            getAccountStatus: jest.fn(() =>
+                Promise.resolve({
+                    get_account_status: { ...mocked_get_account_status },
+                })
+            ),
+        },
+        getSocket: jest.fn(() => Promise.resolve()),
+    },
+}));
+jest.mock('react-router-dom', () => ({
+    ...jest.requireActual('react-router-dom'),
+    Redirect: jest.fn(() => <div>Redirect</div>),
+}));
+
+describe('ProofOfIncome', () => {
+    const files_descriptions_title = 'The document must be recent and include your name and address:';
+    const files_descriptions = [
+        'The document must be up-to-date and signed by the issuance authority.',
+        'The document must contain a letterhead.',
+        'Invalid or incomplete documents shall be rejected.',
+    ];
+    const upload_warning = 'Remember, selfies, pictures of houses, or non-related images will be rejected.';
+    const file_type_requirements = 'Supported formats: JPEG, JPG, PNG, PDF, and GIF only';
+
+    let mock_tore: ReturnType<typeof mockStore>;
+
+    beforeEach(() => {
+        mock_tore = mockStore({});
```

Comment:
```suggestion
        mock_store = mockStore({});
```
nit-pick :) also, it would be great to use existing constant `income_status_codes` for statuses in the test file if possible